### DISCUSSION TITLE
Slash Lighting Behaviour - Customizing Lid-Close Lighting Options?

### DISCUSSION THREAD

**Discussion by kesku** [seerge/g-helper#2920] [Q&A]
Hi there!

I noticed some changes in the recent release regarding the slash lighting settings. Previously, I encountered what seemed like a bug: when I selected one option from the slash lighting dropdown and then switched to another option, the initial option would display when I closed the laptop lid.

I ended up using this "bug" as a feature, setting the lighting to "static" while using the laptop and having it switch to "flux" when I closed the lid and put the machine to sleep. I understand this might not be intended, but is there any way to achieve this behaviour through another method (maybe via `%AppData%\GHelper`)?

Thanks in advance for any help!
![image](https://github.com/user-attachments/assets/089d7066-4fa3-454a-abd8-7c730af47339)

**Comment by seerge** [comment]
@kesku hello, it was a separate animation (built into Slash firmware) that was turning on on lid-close. Check discussion in original issue https://github.com/seerge/g-helper/issues/2815 

I don't want to overcomplicate UI with limitless checkboxes (as majority simply don't understand that). I can try to add it in config tho
